src/components/MainContainer.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from "react";
+import { Task } from "../types/Task";
+import TaskForm from "./TaskForm";
+import TaskList from "./TaskList";
```

Comment:
```suggestion
import { useState } from "react";

import { Task } from "../types/Task";
import TaskForm from "./TaskForm";
import TaskList from "./TaskList";
```
Should add a line break between library and relative imports.